R/checkmate.R

```diff
@@ -233,3 +233,70 @@ test_length <- makeTestFunction(check_length)
 #' @inheritParams check_length
 #' @export
 expect_length <- makeExpectationFunction(check_length)
+
+# assert_range ----
+
+#' Check that an argument is a numerical range
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' An argument `x` is a numerical range if and only if (all conditions must be met): \cr
```

Comment:
I don't think you need all the `\cr` here and below

Reply:
Hmm, but it looks nice with the new lines: 
<img width="675" alt="Screenshot 2022-12-21 at 12 59 34" src="https://user-images.githubusercontent.com/11532997/208900474-55a33792-83dd-4605-9e7c-2cf8e8863349.png">

Reply:
I thought roxygen will handle this automatically, maybe you just need blank lines or so?

Reply:
Ohh, that is true, tx @danielinteractive!